### PR TITLE
Issue#201 teaspoon view path

### PR DESCRIPTION
**Pull request by arenoir**
quick fix to issue #201 add  "lib/ember/appkit/rails/views"  app.view.paths

### CODE REVIEW

**Comment by abuiles** [comment]
@arenoir great! thanks :)

**Comment by abuiles** [comment]
@arenoir btw, travis is not happy

**Comment by arenoir** [comment]
I will look into that.

**Comment by arenoir** [comment]
@abuiles rails 4.0.4 was just released and that is what broke the build (master fails also). I changed the travis ci gemset for now. I will open a new issue #214 .

**Comment by abuiles** [comment]
@bcardarella @rjackson could you guys have a look at this?

File: gemfiles/Gemfile.rails-4.0.x [hunk 1]
```diff
@@ -2,7 +2,7 @@ source "https://rubygems.org"
 
 gemspec path: '../'
 
-gem 'rails', '~>4.0.0'
+gem 'rails', '4.0.3'
```
Comment (bcardarella) [inline gemfiles/Gemfile.rails-4.0.x]: Is there a reason to do the hard coded version?

**Comment by bcardarella** [comment]
I haven't been using teaspoon so I cannot speak to what impact this will have.

**Comment by abuiles** [comment]
@bcardarella I tested and it fixes the issue reported on #201

**Comment by bcardarella** [comment]
Ok, I think the change to the Gemfile is unecessary

**Comment by abuiles** [comment]
@bcardarella cool, seems related with https://github.com/dockyard/ember-appkit-rails/issues/214, @arenoir could you remove the Gemfile change and then we can address the issue with the built in a separate PR?

**Comment by arenoir** [comment]
@abuiles okay reverted the the gemfile.

File: lib/ember/appkit/rails/teaspoon/spec_controller.rb [hunk 2]
```diff
@@ -1,11 +1,5 @@
 module Ember::Appkit::Rails::Teaspoon::SpecController
-  def self.prepended(mod)
```
Comment (abuiles) [inline lib/ember/appkit/rails/teaspoon/spec_controller.rb:2]: @arenoir we don't require this file any more.

**Comment by arenoir** [comment]
I left it in place in case there was some other controller methods that needed to be added. It isn't necessary, I will pull it out.

**Comment by abuiles** [comment]
@arenoir please do, I'm right now looking at the issue with Travis, Rails 4.0.4 broke something.

**Comment by abuiles** [comment]
@bcardarella this seems good to merge, the error on travis is related with a bug on testing generators with Rails 4.0.4, opened issue there.

**Pull request merged**